packages/expo-font/src/FontLoader.web.ts

```diff
@@ -0,0 +1,38 @@
+import ExpoFontLoader from './ExpoFontLoader';
+import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
+import { FontSource, FontResource } from './FontTypes.web';
+/**
+ * A font source can be a URI, a module ID, or an Expo Asset.
+ */
+
+export function fontFamilyNeedsScoping(name: string): boolean {
+  return false;
+}
+
+export function getAssetForSource(source: FontSource): FontResource {
+  if (typeof source === 'object' && 'uri' in source) {
+    return {
+        display: source.display,
+        // @ts-ignore
+        uri: source.uri || source.localUri
```

Comment:
where does `localUri` come from?

Reply:
if someone passes a loaded `Asset`

Reply:
We then either need to add `localUri` to the FontSource type, and fix the `uri in source` check above, or remove this. Right now the program is not clear.